File: Healthcare.netcore/HealthcareAxis.Shared/DTOs/Auth/AuthResponse.cs
namespace HealthAxis.Shared.DTOs.Auth;

public class AuthResponse
{
    public bool Success { get; set; }

    public string UserId { get; set; } = string.Empty;

    public string AccessToken { get; set; } = string.Empty;

    public string RefreshToken { get; set; } = string.Empty;

    public string Message { get; set; } = string.Empty;

    public int ExpiresIn { get; set; }

    public bool RequiresPasswordChange { get; set; } = false;
}

File: HealthApp.Angular/src/app/core/models/auth-response.ts
export interface AuthResponse {
  message: string;
  userId: string;
  email: string;
  role: string;
  accessToken: string;
  refreshToken: string;
  accessTokenExpiresAt: string;
  mustChangePassword: boolean;
  patientId?: number;
  doctorId?: number;
}